src/middlewares/index.ts

```diff
@@ -12,6 +12,9 @@ export const notFoundHandler = function (req: Request, res: Response) {
  * The "next" arg is needed even though it's not used, otherwise express won't understand that it's an error handler
  */
 export function generalErrorHandler(err: any, req: Request, res: Response, next: express.NextFunction) {
+	if (err.errorBody) {
```

Comment:
Because I throw 'throw e' which wrapp the error with body, otherwise the error handler handle it as a 500 error.
The other option was to wrap the 'throw e' with kin bootstrap unknown error.